src/test/java/de/qabel/desktop/ui/contact/ContactControllerTest.java

```diff
@@ -166,7 +167,7 @@ public void importValidContactAfterFailedImport() throws Exception {
         controller.importContacts(c1.toFile());
         try {
             controller.importContacts(c1.toFile());
-            fail("duplicate import failed, cannot continue test");
+            fail("expecting EntityExistsException on duplicate contact");
         } catch (Exception ignored) {}
```

Comment:
good point, 
well then i have to catch out the PersistenceException because EntityExists... is part of it